init/test-postgres.js

```diff
@@ -9,14 +9,14 @@ const run = async () => {
     process.env.TEST_CONTAINER= 'true';
 
     console.log('Running tests...');
-    const program = spawn('nyc', ['playwright', 'test']);
-
-    program.stdout.pipe(process.stdout);
-    program.stderr.pipe(process.stderr);
+    const program = spawn('nyc', ['playwright', 'test'], { stdio: 'inherit', env: process.env });
```

Comment:
Pass env and inherit stdio to make sure github action would be able to read report from logs.